src/main/scala/de/uhh/lt/jst/dt/CoNLL2DepTermContext.scala

```diff
@@ -0,0 +1,88 @@
+package de.uhh.lt.jst.dt
+
+import de.uhh.lt.conll.{CoNLLParser, Row, Sentence}
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+import org.apache.spark.sql.functions.{explode, udf}
+import de.uhh.lt.spark.corpus._
+
+object CoNLL2DepTermContext {
+
+  def main(args: Array[String]): Unit = {
+
+    if (args.length < 2) {
+      println("Usage: input-file output-dir")
```

Comment:
This funcitonality is still not implemented. I will keep that in mind and thank you for pointing out that this will be the main entrypoint. Are you fine with skipping this for now.
Would be nice to have this pull request merged and then continue :)